v/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM thevlang/vlang:debian as build
 
 WORKDIR /app
+RUN v up
```

Comment:
If I understand correctly, `v up` fetch the latest version. Then the version displayed on `config.yaml` will be wrong

Reply:
v still on V 0.4.x

Reply:
for noz, but in the future `v up` could lead to 0.5 right ?

is there any param like `v up -v 0.4` ?

Reply:
also should not be `v up -prod` ?

Reply:
> for noz, but in the future `v up` could lead to 0.5 right ?
> 
> is there any param like `v up -v 0.4` ?

I can change to `latest`. What do you think?

Reply:
> also should not be `v up -prod` ?

No